components/LargeHeader.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+function LargeHeader() {
+  return (
+    <nav className="navbar navbar-default largeheader_background" style={{backgroundImage: 'url(' + 'assets/images/bg.png' +')'}}>
```

Comment:
Agree. Will have to use a loader in webpack.

http://survivejs.com/webpack/loading-assets/loading-images/